src/plugins/chart_expressions/expression_legacy_metric/common/expression_functions/metric_vis_function.ts

```diff
@@ -143,10 +143,6 @@ export const metricVisFunction = (): MetricVisExpressionFunctionDefinition => ({
       if (args.bucket) {
         throw new Error(errors.splitByBucketAndColorFullBackgroundSpecifiedError());
       }
-
-      if (args.metric.length > 1 || input.rows.length > 1) {
```

Comment:
I removed this because this is the case we have for esql and we always pass `colorFullBackground` in Lens. The setting is ignored instead of throwing.